main.test.js

```diff
@@ -1,208 +0,0 @@
-/* eslint-env mocha */
```

Comment:
While you are completely right, that this file should be removed, it has nothing to do with this PR, so please revert it, I'll remove it in the master.